src/runtime/memhash_tsip.go

```diff
@@ -10,11 +10,32 @@
 package runtime
 
 import (
-	"encoding/binary"
-	"math/bits"
```

Comment:
Are you sure you can't use math/bits directly? Using it directly can give some important speed boosts because the compiler treats most of the functions in that package as runtime intrinsics and they often compile down to a single fast instruction.
I checked the package and didn't see an import of any package (except for the unsafe package which itself is a built-in package with no dependencies AFAIK).

Reply:
Looks like the circular import was actually from `encoding/binary` so I've put back the use of `math/bits`.

Reply:
Thanks for the update!
I only now realize I've had this problem before, and fixed it with https://github.com/tinygo-org/tinygo/pull/4306. Can you check whether you can use that internal package instead? (You'll probably need to add `LittleEndian.Uint64`). That would avoid some duplication.

Reply:
`Uint64` added to `internal/binary`, and I switched the `tsip` code over to that: https://github.com/tinygo-org/tinygo/pull/4476/commits/a74be5d833f7af56c33ef0b7dc55545d6518a42f